qt/scientific_interfaces/EnggDiffraction/EnggDiffMultiRunFittingWidgetPresenter.h

```diff
@@ -0,0 +1,51 @@
+#ifndef MANTIDQT_CUSTOMINTERFACES_ENGGDIFFMULTIRUNFITTINGWIDGETPRESENTER_H_
+#define MANTIDQT_CUSTOMINTERFACES_ENGGDIFFMULTIRUNFITTINGWIDGETPRESENTER_H_
+
+#include "DllConfig.h"
+#include "IEnggDiffMultiRunFittingWidgetModel.h"
+#include "IEnggDiffMultiRunFittingWidgetPresenter.h"
+#include "IEnggDiffMultiRunFittingWidgetView.h"
+
+#include <memory>
+
+namespace MantidQt {
+namespace CustomInterfaces {
+
+class MANTIDQT_ENGGDIFFRACTION_DLL EnggDiffMultiRunFittingWidgetPresenter
+    : public IEnggDiffMultiRunFittingWidgetPresenter {
+
+public:
+  EnggDiffMultiRunFittingWidgetPresenter(
+      std::unique_ptr<IEnggDiffMultiRunFittingWidgetModel> model,
+      std::unique_ptr<IEnggDiffMultiRunFittingWidgetView> view);
+
+  void addFittedPeaks(const int runNumber, const size_t bank,
+                      const Mantid::API::MatrixWorkspace_sptr ws) override;
+
+  void addFocusedRun(const int runNumber, const size_t bank,
+                     const Mantid::API::MatrixWorkspace_sptr ws) override;
+
+  boost::optional<Mantid::API::MatrixWorkspace_sptr>
+  getFittedPeaks(const int runNumber, const size_t bank) const override;
+
+  boost::optional<Mantid::API::MatrixWorkspace_sptr>
+  getFocusedRun(const int runNumber, const size_t bank) const override;
+
+  void
+  notify(IEnggDiffMultiRunFittingWidgetPresenter::Notification notif) override;
+
+private:
+  void processShutDown();
```

Comment:
Did you consider renaming these to `start` and `shutdown`. On that note, is there going to be anything done in either of these that couldn't be done in the constructor/destructor?